src/vss_tools/vspec/vspec2x.py

```diff
@@ -43,9 +44,11 @@ def __init__(self, generator: Vss2X, vspec2vss_config: Vspec2VssConfig):
         self.vspec2vss_config = vspec2vss_config
 
     def main(self, arguments):
-        initLogging()
         parser = argparse.ArgumentParser(description="Convert vspec to other formats.")
 
+        parser.add_argument("--log-level", choices=["INFO", "DEBUG", "WARNING", "ERROR", "CRITICAL"],
```

Comment:
Could you take a look at https://github.com/COVESA/vss-tools/blob/master/docs/vspec2x.md and add some info there as well on the new arguments.

(And yes - I know that page is a bit messy so if feel free to more than just adding the two new args there if you like :-) )

Reply:
Sure, will do

Reply:
Done. Tried to clean up and modernize all docs in the process.